misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/distribution/DistributionStepScreen.jsx

```diff
@@ -104,6 +107,20 @@ const DistributionStepScreen = () => {
     );
   };
 
+  const onUnpick = () => {
+    postDistributionReversePicking({
+      wfProcessId,
+      activityId,
+      lineId,
+      stepId,
+    })
+      .then((wfProcess) => {
+        history.goTo(distributionLineScreenLocation);
```

Comment:
why not history.goBack() ?

Reply:
I wanted to make sure that I always end up on the line screen, so the user can easily see the unpicked step is gone.

Reply:
@cp-ps ping